src/handler.js

```diff
@@ -1,8 +1,42 @@
 const { framework, router } = require('@ewarren/serverless-routing');
+const mongodb = require('mongodb');
 const AWS = require('aws-sdk');
 
 const s3 = new AWS.S3();
 const app = framework({ basePath: '/:stage-events' });
+const EventModel = require('models/Event');
+
+let cachedDb = null;
+
+function connectToDatabase() {
```

Comment:
Not at all a blocker for this PR; but something I was thinking about when I added this to `slack-for-sms` was the best way to package this bit of code so it can be more easily re-used across projects that require Mongo.

A/ New package, eg: `serverless-mongo-connect` or something (where I'm leaning atm)
B/ Add it to `serverless-routing` as a utility export.

Either way, just a cleanup task for later, no need to worry about it now.

Reply:
Yeah I agree. How about when we set up the third node-based lambda method, we refactor stuff that would be triplicated across all three into one common util library for node-based lambda methods.